docs/source/model_design/vivarium_overview/data_sources/index.rst

```diff
@@ -1,26 +1,157 @@
+
+.. list-table:: Sources of draws
+   :header-rows: 1
+
+   *  - Source
+      - Description
+      - GBD ID type
+      - Note
+   *  - :code:`epi`
+      - Dismod and custom epi models. This source contains data that is computed by GBD modelers and often used as inputs to central GBD processes.
+      - modelable_entity_id
+      - 
+   *  - :code:`codcorrect`
+      - Deaths and YLLs
+      - cause_id
+      - Returns **counts** only
+   *  - :code:`como`
+      - YLDs, incidence, and comorbidity-adjusted prevalence
+      - cause_id, sequela_id, rei_id
+      - Returns **rates** only
+   *  - :code:`dalynator`
+      - DALYs
+      - cause_id
+      - 
+   *  - :code:`exposure`
+      - Risk factor exposure
+      - rei_id
+      - Can be continuous (like mean BMI) or categorical (like stunting prevalence)
+   *  - :code:`exposure_sd`
+      - Risk exposure standard deviation
+      - rei_id
+      - Only for continuous risks
+   *  - :code:`rr`
+      - Risk factor relative risk
+      - rei_id, cause_id
+      - Will return values for all affected causes unless a cause_id is specified
+   *  - :code:`burdenator`
+      - Risk attributable burden (deaths/dalys/ylls/ylds) and mediated/aggregated PAFs
+      - cause_id, rei_id
+      - 
+   *  - :code:`paf`
+      - Pre-burdenator (non-finalized) PAF estimates
+      - rei_id, cause_id
+      - 
+   *  - :code:`sev`
+      - Summary exposure values
+      - rei_id
+      - 
+   *  - :code:`tmrel`
+      - Risk factor theoretical minimum exposure level
+      - rei_id
+      - 
+   *  - :code:`rr_max`
+      - Relative risk maximum value
+      - rei_id, cause_id
+      - 
+   *  - :code:`codem`
+      - codem models and custom cod models
+      - cause_id
+      - 
+   *  - :code:`stgpr`
+      - ST-GPR models
+      - modelable_entity_id
+      - If you pass an MEID with a dismod model type but try to use the ST-GPR source, get_draws will use the epi source instead.
+
+Handling GBD versioning
+++++++++++++++++++++++++
+
+Decomposition (or "decomp") steps are a versioning scheme used in some GBD rounds that allowed updates to GBD results based on iterative updates to certain parts of the computation process. For instance, the first step may be equivalent to the prior GBD round in all aspects except for an updated demographic model; the second step may be equivalent to the prior steps, but with updated risk exposures; and so on. This process allowed GBD researchers to evaluate individual components of the many changes included in a GBD round advancement influenced the main results of the GBD study rather than updating the entire pipeline at once. 
```

Comment:
```suggestion
Decomposition (or "decomp") steps are a versioning scheme used in some GBD rounds that allowed updates to GBD results based on iterative updates to certain parts of the computation process. For instance, the first step may be equivalent to the prior GBD round in all aspects except for an updated demographic model; the second step may be equivalent to the prior steps, but with updated risk exposures; and so on. This process allowed GBD researchers to evaluate how individual components of the many changes included in a GBD round advancement influenced the main results of the GBD study, rather than updating the entire pipeline at once.
```

I think this sentence is missing a word, but I'm not sure if this is the right one :)